git-tips-for-beginners-part-2.adoc

```diff
@@ -0,0 +1,136 @@
+== Git: советы новичкам – часть 2
+^Блог^ ^компании^ ^Playrix,^ ^Git^
+
+Во второй части нашего пособия для новичков в Git мы рассказываем об управлении ветками, особенностях их слияния, а также о принципах работы указателей. Первую статью вы можете прочитать по https://habr.com/ru/company/playrix/blog/345732/[ссылке].
+
+image::img/git text 1.png[]
```

Comment:
Remove it!

Reply:
should I remove just picture or picture and text above?